test/runtests.jl

```diff
@@ -345,6 +374,7 @@ end
         callsite = only(callsites)
         info = callsite.info
         @test isa(info, Cthulhu.InvokeCallInfo)
+        Cthulhu.EFFECTS_ENABLED && @test_broken Cthulhu.get_effects(info) |> Core.Compiler.is_total  # don't have effects from InvokeCallInfo yet
```

Comment:
Can't we try to get effects from `Core.invoke` callsite? Now I think we are just passing `Effects()` explicitly in such case.
https://github.com/JuliaDebug/Cthulhu.jl/blob/8905460b785a67dbfcb759a37687429756f3b9ec/src/reflection.jl#L157